torntools/scripts/js/globalFunctions.js

```diff
@@ -1585,7 +1585,7 @@ const STORAGE = {
 				worth: false,
 			},
 			attack: {
-				warn_when_stacking: true,
+				warn_when_stacking: true, // Debate: false by default
```

Comment:
But that is a useful feature and I have seen many ppl do training as part of their regular schedule forgetting that they are stacking or attack some guy who disturbed them and regret in faction chat. Hence, I think it should be enabled by default.

Reply:
It's an intrusive feature though. Most people who stack do it for chains/wars, so showing it on attacks by default might not be the best option.

Reply:
Hence I did this commit #371 .

Reply:
Someone has to start the chain though.

Reply:
Yea, I agree to turn it off by default. Have had some messages in Torn about it too.